src/shadowbox/server/manager_service.ts

```diff
@@ -616,8 +616,48 @@ export class ShadowsocksManagerService {
 
   async getDataUsage(req: RequestType, res: ResponseType, next: restify.Next) {
     try {
-      logging.debug(`getDataUsage request ${JSON.stringify(req.params)}`);
-      const response = await this.managerMetrics.getOutboundByteTransfer({hours: 30 * 24});
+      logging.debug(
+        `getDataUsage request ${JSON.stringify(req.params)} ${JSON.stringify(req.query)}`
+      );
+
+      // Default to 30 days (30 * 24 hours) for backward compatibility
+      const DEFAULT_HOURS = 30 * 24;
```

Comment:
I think we have a duration type. We should be handling duration objects instead.